app/src/main/res/layout/status_bar.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--suppress AndroidDomInspection -->
 <View xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/status_bar"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/status_bar_padding"
+    android:layout_height="@*android:dimen/status_bar_height"
```

Comment:
@kabouzeid I just noticed that I forgot to delete this file, I think I deleted all usages and replaced them with fitSystemWindows. This xml can probably be deleted.